Makefile

```diff
@@ -1,8 +1,12 @@
 .PHONY: all
 all:
 
+.PHONE: test-all
+test-all: checkstyle test
+
 .PHONY: test
-test: checkstyle
+test:
+	prove test/*
```

Comment:
`prove -r` should work

Reply:
I'm guessing that's slower if it needs to search the whole tree?

Reply:
I made it `prove -r test/` now.